packages/react-divider/src/components/Divider/useDivider.ts

```diff
@@ -1,36 +1,42 @@
 import * as React from 'react';
-import { makeMergeProps, resolveShorthandProps, useId, useMergedRefs } from '@fluentui/react-utilities';
-import { DividerProps, DividerState } from './Divider.types';
+import { getNativeElementProps, resolveShorthand, useId } from '@fluentui/react-utilities';
+import { DividerProps, DividerSlots, DividerState } from './Divider.types';
 
 /**
- * Consts listing which props are shorthand props.
+ * Const listing which props are shorthand props.
  */
-export const dividerShorthandProps = ['wrapper', 'children'] as const;
-
-const mergeProps = makeMergeProps<DividerState>({ deepMerge: dividerShorthandProps });
+export const dividerShorthandProps: Array<keyof DividerSlots> = ['root', 'wrapper'];
```

Comment:
This can be inlined in `renderDivider` now as it is the only place where it is used.